packages/tools/cookbook/src/accounts/transfer-create.ts

```diff
@@ -49,22 +47,20 @@ async function transferCreate(
     .addCap('coin.TRANSFER', senderPublicKey, sender, receiver, amount)
     .setMeta({ sender }, NETWORK_ID);
 
-  const signedTransactions = await signWithChainweaver(transactionBuilder);
+  await signWithChainweaver(transactionBuilder);
 
-  const sendRequests = signedTransactions.map((tx) => {
-    console.log(`Sending transaction: ${tx.code}`);
-    return tx.send(API_HOST);
+  await transactionBuilder.send(API_HOST);
```

Comment:
(for my own education)
I noticed we're no longer keeping a reference to the `signedTransactions`. Does the `transactionBuilder` keep a copy of the signed transactions in its state now?

Reply:
Nothing changed in the `transactionBuilder`, we just use it differently in the cookbook now. There was no need to keep track of multiple transactions here, since there is only one. Line 52 awaits the transaction to succeed. When you reject the transaction in Chainweaver or it fails in another way it will[ throw an error](https://github.com/kadena-community/kadena.js/blob/master/packages/libs/client/src/signing/signWithChainweaver.ts#L85).